src/v/cluster/rm_stm.cc

```diff
@@ -2076,13 +2076,21 @@ ss::future<> rm_stm::do_abort_old_txes() {
     }
 }
 
+ss::future<> rm_stm::force_abort_tx(model::producer_identity pid) {
```

Comment:
no need to change anything in rm_stm, it already has mark_expired which does exactly the same, just invoke it

Reply:
Oh. you are right! Thanks